fapolicy_analyzer/tests/test_subject_list.py

```diff
@@ -339,6 +339,8 @@ def test_right_click_menu_and_select(subjects, widget):
     event = Gdk.EventButton
     event.type = Gdk.EventType.BUTTON_PRESS
     event.button = 3
+    event.x = 210
+    event.y = 7
```

Comment:
Pretty sure this is not needed in this PR. I think it came from #484.  Let me know if I'm wrong on that one, but this change exists in both branches.